src/tests/api/BavHappyPathTest.test.ts

```diff
@@ -15,18 +15,21 @@ describe("Test BAV End Points", ()=> {
         //Session Request
         const stubResponse = await startStubServiceAndReturnSessionId(bavStubPayload);
         sessionId = stubResponse.data.session_id;
+        console.log("Session Id: " + sessionId);
```

Comment:
If `sessionID` is all we need shall we just return that from the `startStubServiceAndReturnSessionId` function rather than the whole object?

Reply:
Updated so startStubServiceAndReturnSessionId returns sessionId 👍